hbase-common/src/main/java/org/apache/hadoop/hbase/util/CommonFSUtils.java

```diff
@@ -1,4 +1,4 @@
-/*
+/**
```

Comment:
Why in the above xml we change ** to * and here we change * to **...

Reply:
> Why in the above xml we change ** to * and here we change * to **...

This reverts the two existing commits. I have not retained the minor improvements they provided.